API/src/main/java/de/brandwatch/minianalytics/api/controller/QueryController.java

```diff
@@ -0,0 +1,63 @@
+package de.brandwatch.minianalytics.api.controller;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import de.brandwatch.minianalytics.api.service.QueryService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+
+@RestController
+public class QueryController {
+
+    private static final Logger logger = LoggerFactory.getLogger(QueryController.class);
+
+    private QueryService queryService;
+
+    @Autowired
+    public QueryController(QueryService queryService) {
+        this.queryService = queryService;
+    }
+
+    private static final Gson gson = new GsonBuilder().setPrettyPrinting().create();
+
+    @PostMapping(value = "/queries")
+    public ResponseEntity query(@RequestParam String query){
+        try {
+            logger.info("POST API Call: /query");
+            logger.info("Query: " + query);
```

Comment:
I'd suggest to put these into one log. Also, never concatenate like this, instead, do 
```suggestion
            logger.info("Query: {}", query);
```

I'd suggest:

```
logger.info("POST /query: {}", query);
```

or if you want also something like this:

```
logger.info("'POST /query' api call with query string '{}'", query);
```

Reply:
I reformatted the logger outputs.